packages/core/types/src/http/product/common.ts

```diff
@@ -0,0 +1,192 @@
+import { BaseFilterable, OperatorMap } from "../../dal"
+import { BaseProductCollection } from "../collection/common"
+
+export type ProductStatus = "draft" | "proposed" | "published" | "rejected"
```

Comment:
todo: unrelated, but I ve seen that we have an enum for that in the types package and it shouldnt be there if we want to clean it up :) instead it should a type and you can maybe move this one in the place the enum is located in the product type. wdyt?

Reply:
@adrien2p yeah I will move things around once you're done with the cleanup

Reply:
👍 It is in the same package, but in the product directory